fury/shaders/base.py

```diff
@@ -170,3 +170,60 @@ def attribute_to_actor(actor, arr, attr_name, deep=True):
     mapper = actor.GetMapper()
     mapper.MapDataArrayToVertexAttribute(
         attr_name, attr_name, vtk.vtkDataObject.FIELD_ASSOCIATION_POINTS, -1)
+
+
+class Uniform:
+    def __init__(self, name, type, value):
+        """
+        Parameters:
+        -----------
+            name: str
+            type: str
+            value: float or ndarray
+        """
+        self.name = name
+        self.value = value
+        self.type = type
+        self.vtk_func_uniform = ''
+        self.setType()
+
+    def setType(self):
```

Comment:
1 - Please, python convention: `set_type`
2 - Strange to have `set_type` without argument / the new type.